scripts/qa_crowbarsetup.sh

```diff
@@ -1000,6 +1000,10 @@ function onadmin_prepare_sles12sp1_other_repos()
     for repo in SLES12-SP1-{Pool,Updates}; do
         add_mount "$repo/sle-12-x86_64" "$clouddata:/srv/nfs/repos/$repo" \
             "$tftpboot_repos12sp1_dir/$repo"
+        if [[ $want_s390 ]] ; then
```

Comment:
yes, but we have that same behaviour in many other places as well, where we test  with [ -n $boolean ] or [[ ]]